### PR TITLE
Tools - README Updates

### PR DESCRIPTION
**Pull request by ford-at-aws**
This PR updates the instructions for testing and specifically adding a new tag/release.

Pending improvements whereby automation might do this for us via GH Actions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,17 +73,39 @@ Some validation options can be extended by creating `.doc_gen/validation.yaml`.
...
-  * Also add that commit sha to the [validation config](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/.doc_gen/validation.yaml)
-3. Create a PR in aws-doc-sdk-examples (do not submit it) to run tests.
-4. When tests in the main repo are passing, create a tag in this -tools repo at that sha with the format `YYYY-MM-DD-A`, where `YYYY-MM-DD` are the year, month, and day of the release and -A is -A, -B, -C etc for multiple releases within a day.
-5. Push that tag.
-6. Update the PR from 2 with that tag, remove the validation sha exception, and push that for draft.
-7. Create a release using the automated "Create release from tag" button.
-8. Perform internal update process.
+### 1. Testing
 
+1. **Create a testing branch** from [aws-doc-sdk-examples:main](https://github.com/awsdocs/aws-doc-sdk-examples).
```
Comment (DavidSouther) [inline README.md:80]: Ref should be awsdocs/aws-doc-sdk-examples@main (and below)

File: README.md [hunk 2]
```diff
@@ -73,17 +73,39 @@ Some validation options can be extended by creating `.doc_gen/validation.yaml`.
...
+
+1. **Find the most recent commit SHA in [aws-doc-sdk-examples-tools:main](https://github.com/awsdocs/aws-doc-sdk-examples-tools/commits/main/)**.
+
+1. **Update your testing branch**: Add your SHA to the following files in tag format (e.g. `awsdocs/aws-doc-sdk-examples-tools@e7c283e916e8efc9113277e2f38c8fa855a79d0a`):
```
Comment (DavidSouther) [inline README.md:84]: What is a "tag format"?

File: README.md [hunk 3]
```diff
@@ -73,17 +73,39 @@ Some validation options can be extended by creating `.doc_gen/validation.yaml`.
...
+1. **Open a Draft PR to main branch**: Do not publish for review. Wait for checks/tests to pass on the PR.
+
+### 2. Deployment
+
+1. **Create a -tools tag**: Once the tests pass, create a tag in the -tools repository at the same SHA you identified earlier.
+   - NOTE: tag format is `YYYY-MM-DD-A`, where `YYYY-MM-DD` represents release date, and `-A` is used for the first release of the day (followed by `-B`, `-C`, etc., for subsequent same-day releases)
+
+Here is a command line example, tested on Mac:
+```
+TAG_NAME=$(date +%Y-%m-%d)-A && SHA=$(git rev-parse HEAD) && git tag -a "$TAG_NAME" "$SHA" -m "Release $TAG_NAME" && git push origin "$TAG_NAME"
```
Comment (DavidSouther) [inline README.md:98]: Line breaks to make this readable

**Review by DavidSouther — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ford-at-aws) [reply README.md:84]: Probably wrong wording. I mean this format: `org/repo@<tag>`. Suggestion: "`org/repo@tag` format"

File: README.md [hunk 4]
```diff
@@ -73,17 +73,33 @@ Some validation options can be extended by creating `.doc_gen/validation.yaml`.
...
-3. Create a PR in aws-doc-sdk-examples (do not submit it) to run tests.
-4. When tests in the main repo are passing, create a tag in this -tools repo at that sha with the format `YYYY-MM-DD-A`, where `YYYY-MM-DD` are the year, month, and day of the release and -A is -A, -B, -C etc for multiple releases within a day.
-5. Push that tag.
-6. Update the PR from 2 with that tag, remove the validation sha exception, and push that for draft.
-7. Create a release using the automated "Create release from tag" button.
-8. Perform internal update process.
+### 1. Testing
 
+1. **Create a testing branch** from [aws-doc-sdk-examples@main](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main).
+2. **Find the most recent commit SHA in [aws-doc-sdk-examples-tools:main](https://github.com/awsdocs/aws-doc-sdk-examples-tools/commits/main/)**.
```
Comment (DavidSouther) [inline README.md:81]: (nit) same

File: README.md [hunk 5]
```diff
@@ -73,17 +73,33 @@ Some validation options can be extended by creating `.doc_gen/validation.yaml`.
...
+3. **Update your testing branch**: Add your SHA as a tag (`org/repo@tag`, e.g. `awsdocs/aws-doc-sdk-examples-tools@e7c283e916e8efc9113277e2f38c8fa855a79d0a`) to the following files:
```
Comment (DavidSouther) [inline README.md:82]: "Add your sha as a tag" this doesn't make sense? There are no tags in this step? Tags are created in step 1 part 2

**Review by DavidSouther — CHANGES_REQUESTED** [review]

**Review by DavidSouther — APPROVED** [review]

**Pull request merged**